bin/studio-common

```diff
@@ -38,6 +39,17 @@ function document() {
   [ -z "$CI" ] && echo -e "$content" > "/tmp/docs/${name}"
 }
 
+# This is a helper function that allows you to quickly and easily add an alias for
+# a helper function. It will also ensure that a list of aliases are printed when
+# you run `describe`.
+function add_alias() {
+  fun="$1"
+  a="$2"
```

Comment:
maybe some protection? (what if they don't provide any argument or just one?)

Reply:
Sure. What type of behavior would you expect?

Reply:
Not needed honestly but I always ask myself questions like;
* What if they don't specify one argument and I eval `alias foobear=` or none `alias =` (I think one of them will blow up)
* What if I set the same alias to two functions? I would like to get a warning or block the action.